test/docker.js

```diff
@@ -0,0 +1,14 @@
+const exec = require('child_process').exec;
```

Comment:
could makefile include installation of docker if not found. I guess it could be useful.